### ISSUE TITLE
some questions

### ISSUE THREAD

**Issue by jiangzhengkai** [sciencefans/Focal-Loss#8]
when calculate average loss,why not ignore the number of ignored label?(loss = loss/batch_size is not correct?)Thank you！

**Comment by liuyuisanai** [comment]
Please refer to `get_normalizer()` and `LossParameter_NormalizationMode_VALID` in line51 of .cpp and line88-89 of .cu.
